Frontend/CO.CDP.OrganisationApp/Pages/Forms/FormElementUrlInputModel.cs

```diff
@@ -25,7 +25,7 @@ public class FormElementUrlInputModel : FormElementModel, IValidatableObject
         if (HasValue != false && !string.IsNullOrWhiteSpace(TextInput))
         {
             formAnswer ??= new FormAnswer();
-            formAnswer.TextValue = TextInput.Trim();
+            formAnswer.TextValue = TryFixUrl(TextInput);
         }
```

Comment:
Do we need to keep the trimming? It looks like we lost that feature here?

Reply:
Should we? .Net Url object successfully validates the url

Reply:
Extra spaces will get passed on through the data sharing API though? I think we should be a good citizen and trim it. Like the "be tolerant in what you receive but strict in what you send" adage

Reply:
Done :)